src/golang/cmd/executor/executor/log.go

```diff
@@ -0,0 +1,33 @@
+package executor
```

Comment:
I'm wondering if the log file location will be shared across the stack, outside of executor. In that case I would consider moving it to `/lib`

Reply:
Moved the remaining helper to main.py. I think we can refactor this to /lib if we find another use case.